tests/unit/CertificateGenerationService.unitTest.ts

```diff
@@ -0,0 +1,161 @@
+import {expect} from "chai";
+import {CertificateGenerationService} from "../../src/services/CertificateGenerationService";
+import sinon from "sinon";
+import techRecordResp from "../resources/tech-records-response.json";
+import {AWSError, Lambda, Response} from "aws-sdk";
+import {LambdaService} from "../../src/services/LambdaService";
+
+describe("Certificate Generation Service", () => {
+  const sandbox = sinon.createSandbox();
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe("getVehicleMakeAndModel function", () => {
```

Comment:
Didn't go with Cucumber format for several reasons:
1) this was fixing a bug, not a feature, so shouldn't have ACs. Even If it did, ACs wouldn't cover the necessary test scenarios.
2) This entire project is so far behind our "ideal testing structure" that jest-cucumber is not landing any time soon. Bigger fish to fry - Jest, remove sinon, remove chai etc.